apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/BaseStressTest.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Google.Cloud.Spanner.V1;
+using Google.Cloud.Spanner.V1.Logging;
+
+namespace Google.Cloud.Spanner.Data.IntegrationTests
+{
+    public abstract class BaseStressTest
+    {
+        protected const int TargetQps = 100;
+        protected static readonly TimeSpan TestDuration = TimeSpan.FromSeconds(60);
+
+        protected abstract Task<TimeSpan> TestWrite1(Stopwatch sw);
```

Comment:
I started out as having a method to writeX which called this to write "1".
I think TestWriteOneRow is more descriptive, but should probably be spelled out versus a number.  let me know if you disagree.